README.md

```diff
@@ -6,16 +6,59 @@ Call the Harness.io API to deploy an application.
 
 ## Usage
 
+```yaml
+   - name: Deploy
+     id: deploy
+     uses: fac/harness-deploy-action@v1
+     with:
+       webhookUrl: ${{ secrets.HARNESS_WEBHOOK_URL }}
+       application: XYZ12345678ABC
+       services: web,console,job
+       version: ${{ github.sha }}
+
+   - name: Deploy Link
+     run: |
+       echo harness_url: ${{ steps.deploy.outputs.harness_url }}
+```
+
 ## Inputs
 
+### webhookUrl
+
+The full webhook url for harness, including the authentication token. You **should** pass this as a secret.
+
+e.g. `https://app.harness.io/gateway/api/webhooks/XXXXXXXX?accountId=YYYYYYYYYY`, where `XXXXXXXX` and `YYYYYYYY` are actual tokens. You can get this from the harness UI.
```

Comment:
I feel like describing the format of the URL isn't as useful as explaining how to set up a webhook trigger with a custom payload & copying the manual link

the Harness docs are… not very clear on how to do it: https://docs.harness.io/article/ys3cvwm5gc-trigger-a-deployment-on-git-event#step_1_add_a_trigger
Use a 'Custom' Payload type